tests/codegen/params_count.rs

```diff
@@ -0,0 +1,17 @@
+#![feature(no_core)]
+#![no_core]
+#![no_main]
+
+extern crate mini_core;
+
+// expect three int params
+// CHECK: {{((int32_t .*,?\s?){3})}}
+#[no_mangle]
+pub fn foo(_x: i32, _y: i32, _z: i32) -> i32 {
```

Comment:
You probably want to check the full signature? Something like

```rust
// CHECK: int32_t foo(int32_t {{\w+}}, int32_t {{\w+}}, int32_t {{\w+}}) {
// CHECK-NEXT: return 0;
#[no_mangle]
pub fn foo(_x: i32, _y: i32, _z: i32) -> i32 {
```

Since filecheck only checks for partial matches .